examples/pytorch/vision/Face_Detection/data/config_qvga.py

```diff
@@ -54,11 +54,12 @@
 _C.USE_NMS = True
 
 # dataset config
-_C.HOME = '/mnt/'
+_C.HOME = '/mnt'
```

Comment:
Is it possible to incorporate this into the `prepare_wider_data.py` as a runtime argument? That would make it easier to run from inside the jupyter notebook.